OpenEdXMobile/src/main/java/org/edx/mobile/view/WebViewDiscoverFragment.java

```diff
@@ -60,14 +61,23 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             searchQueryExtra = getArguments().getString(Router.EXTRA_SEARCH_QUERY);
         }
 
+        initURL();
+        binding.swipeContainer.setOnRefreshListener(() -> {
+            initURL();
+            binding.swipeContainer.setRefreshing(false);
+        });
+        UiUtils.INSTANCE.setSwipeRefreshLayoutColors(binding.swipeContainer);
+        if (!EventBus.getDefault().isRegistered(this)) {
+            EventBus.getDefault().register(this);
+        }
+    }
+
+    private void initURL() {
```

Comment:
**Issue Steps:** 

1. Search 'Python' from `LunchDiscoverActivity`
2. On discover webview, search for 'Intro'
3. Now Perform Swipe Refresh
4. The search will again be set to 'Python'

We should update `searchQueryExtra` with query params of `binding.webview.getUrl()` if the URL is valid.